contracts/broadcast/FeeContract.s.sol/11155111/run-1723219248.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
For my own understanding, Are these run files used by something in the repo like CI or tests?

If we do need to be committing them to the repo, what are we using them for? If not should we add the `/contracts/broadcast` directory to the .gitignore and remove ones that currently reside in the repo?

Reply:
The idea is to only commit the files for public deployments to the repo. It's to make life easy for anyone who wants to interact with the deployed contracts.